src/encoded/static/components/search.js

```diff
@@ -552,52 +554,128 @@ const DatasetComponent = (props, reactContext) => {
     let biosampleTerm;
     let organism;
     let lifeSpec;
+    let assays = [];
     let targets;
     let lifeStages = [];
     let ages = [];
+    let ageUnits;
+
+    let treatmentTerm = [];
     let treatments = [];
+    let treatmentUnit;
+
+    let organisms = [];
+    let fullStages = [];
+    let phases = [];
+    let synchronization;
+    let postSynchronizationTimeUnits;
+    let postSynchTime = [];
+
+    const haveSeries = result['@type'].indexOf('Series') >= 0;
+    const haveFileSet = result['@type'].indexOf('FileSet') >= 0;
+    const treatmentTime = result['@type'].indexOf('TreatmentTimeSeries') >= 0;
+    const treatmentConcentration = result['@type'].indexOf('TreatmentConcentrationSeries') >= 0;
+    const showTreatment = treatmentTime || treatmentConcentration;
+    const organismSeries = result['@type'].indexOf('OrganismDevelopmentSeries') >= 0;
 
     // Determine whether the dataset is a series or not
     const seriesDataset = result['@type'].indexOf('Series') >= 0;
 
     // Get the biosample info for Series types if any. Can be string or array. If array, only use iff 1 term name exists
     if (seriesDataset) {
         biosampleTerm = (result.biosample_ontology && Array.isArray(result.biosample_ontology) && result.biosample_ontology.length === 1 && result.biosample_ontology[0].term_name) ? result.biosample_ontology[0].term_name : ((result.biosample_ontology && result.biosample_ontology.term_name) ? result.biosample_ontology.term_name : '');
-        const organisms = (result.organism && result.organism.length > 0) ? _.uniq(result.organism.map(resultOrganism => resultOrganism.scientific_name)) : [];
-        if (organisms.length === 1) {
-            organism = organisms[0];
-        }
 
         // Dig through the biosample life stages and ages
         if (result.related_datasets && result.related_datasets.length > 0) {
             result.related_datasets.forEach((dataset) => {
+                if (dataset.assay_term_name) {
+                    assays.push(dataset.assay_term_name);
+                }
                 if (dataset.replicates && dataset.replicates.length > 0) {
                     dataset.replicates.forEach((replicate) => {
                         if (replicate.library && replicate.library.biosample) {
                             const biosample = replicate.library.biosample;
                             const lifeStage = (biosample.life_stage && biosample.life_stage !== 'unknown') ? biosample.life_stage : '';
+                            if (biosample.life_stage !== 'unknown' && biosample.life_stage && biosample.age_display) {
+                                const fullStage = `${biosample.life_stage[0].toUpperCase()}${biosample.age_display.split(' ')[0]}`;
```

Comment:
I did this
`var i = []; i.push('a b'.split(' '))` // the use of var is long tangential story
Is the what you intended to do in line 602?

Reply:
I don't think so. This is for constructing `Stages: E10.5, E11.5, E14.5` on the mouse development search result.